rules/gravitational_teleport_rules/teleport_long_lived_certs.py

```diff
@@ -0,0 +1,76 @@
+from datetime import timedelta, datetime
+from typing import Dict, Tuple
+
+from panther_teleport_helpers import (
```

Comment:
I think these are missing?

Reply:
Indeed, these _are_ missing. I will break these out into `panther_default` in the other non-`panther_config` PR.